File: app/src/main/java/github/xtvj/cleanx/utils/AppUtils.java
package github.xtvj.cleanx.utils;


import static github.xtvj.cleanx.utils.LogExtKt.log;

public class AppUtils {

    private AppUtils() {
        throw new UnsupportedOperationException("u can't instantiate me...");
    }

    public static Boolean isAppRoot() {
        ShellUtils.CommandResult result = ShellUtils.execCmd("echo root", true);
        log(result.toString());
        return result.result == 0;
    }
}

File: app/src/main/java/github/xtvj/cleanx/utils/AppUtils.kt
package github.xtvj.cleanx.utils

class AppUtils private constructor() {
    companion object {
        val isAppRoot: Boolean
            get() {
                val result = ShellUtils.execCmd("echo root", true)
                log(result.toString())
                return result.result == 0
            }
    }

    init {
        throw UnsupportedOperationException("u can't instantiate me...")
    }
}